From f775a30ba3a91ecbd6c4035ce2b08d8fcdc891e7 Mon Sep 17 00:00:00 2001
From: Jo Shields <joshield@microsoft.com>
Date: Fri, 2 Aug 2019 14:05:12 -0400
Subject: [PATCH] Revert "configure.ac: Fix automake suggestion about
 AC_CONFIG_MACRO_DIRS and ACLOCAL_AMFLAGS"

This reverts commit 410d4e3009f744fc29d1ed15a11b80cc72909dd8.
---
 .gitignore   | 5 -----
 Makefile.am  | 2 --
 configure.ac | 3 +--
 3 files changed, 1 insertion(+), 9 deletions(-)

diff --git a/.gitignore b/.gitignore
index 1bf406e..fc2dbb5 100644
--- a/.gitignore
+++ b/.gitignore
@@ -14,11 +14,6 @@ ltmain.sh
 missing
 aclocal.m4
 autom4te.cache/
-libtool.m4
-lt~obsolete.m4
-ltoptions.m4
-ltsugar.m4
-ltversion.m4
 config.status
 libtool
 stamp-h1
diff --git a/Makefile.am b/Makefile.am
index f40bdee..0ac9e50 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,3 @@
-ACLOCAL_AMFLAGS = -I m4
-
 SUBDIRS = src tests
 DIST_SUBDIRS = src tests
 
diff --git a/configure.ac b/configure.ac
index 25fb592..6f48f72 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,8 +1,7 @@
 AC_INIT(libgdiplus, [6.0],
 	[https://github.com/mono/mono/issues/new])
+	AC_CONFIG_SRCDIR([README.md])
 
-AC_CONFIG_SRCDIR([README.md])
-AC_CONFIG_MACRO_DIRS([m4])
 AC_CANONICAL_SYSTEM
 AC_CANONICAL_HOST
 
-- 
2.17.1

